package.json

```
@@ -1813,6 +1813,155 @@
+														"command": "editor.action.formatSelection.multiple"
+													}
+												]
+											},
+											{
+												"key": "b",
+												"name": "+Backend",
+												"type": "bindings",
+												"bindings": [
+													{
+														"key": "d",
+														"name": "Reset Database",
+														"type": "command",
+														"command": "C_Cpp.ResetDatabase"
+													},
+													{
+														"key": "w",
+														"name": "Rescan Workspace",
+														"type": "command",
+														"command": "C_Cpp.RescanWorkspace"
+													}
+												]
+											},
+											{
+												"key": "g",
+												"name": "+Go to",
+												"type": "bindings",
+												"bindings": [
+													{
+														"key": "a",
+														"name": "Switch Header/Source",
+														"type": "command",
+														"command": "C_Cpp.SwitchHeaderSource"
+													},
+													{
+														"key": "d",
+														"name": "Peek declaration",
+														"type": "command",
+														"command": "editor.action.peekDeclaration"
+													},
+													{
+														"key": "e",
+														"name": "Go to errors/problems",
+														"type": "command",
+														"command": "workbench.actions.view.problems"
+													},
+													{
+														"key": "f",
+														"name": "Go to file in explorer",
+														"type": "command",
+														"command": "workbench.files.action.showActiveFileInExplorer"
+													},
+													{
+														"key": "g",
+														"name": "Peek definition",
+														"type": "command",
+														"command": "editor.action.peekDefinition"
+													},
+													{
+														"key": "r",
+														"name": "Peek references",
+														"type": "command",
+														"command": "editor.action.referencesSearch.trigger"
+													},
+													{
+														"key": "s",
+														"name": "Go to symbol in editor",
+														"type": "command",
+														"command": "workbench.action.gotoSymbol"
+													},
+													{
+														"key": "D",
+														"name": "Go to declaration",
+														"type": "command",
+														"command": "editor.action.revealDeclaration"
+													},
+													{
+														"key": "G",
+														"name": "Go to definition",
+														"type": "command",
+														"command": "editor.action.revealDefinition"
+													},
+													{
+														"key": "R",
+														"name": "Find all references",
+														"type": "command",
+														"command": "references-view.findReferences"
+													}
+												]
+											},
+											{
+												"key": "r",
+												"name": "+Refactor",
+												"type": "bindings",
+												"bindings": [
+													{
+														"key": "s",
```

```suggestion
													{
														"key": "r",
```
Other language uses `r`. Probably good to follow?

---

Sure. I'll make this change.